examples/rocket-0_4-hello/README.md

```diff
@@ -4,7 +4,7 @@ This is a hello world `rocket 0.4.x` version example, using `utoipa` and `utoipa
 
 If you are looking for rocket `0.5.x` example look for [rocket-todo](../rocket-todo).
 
-Just run command below to run the demo application and browse to `http://localhost:8000/swagger-ui/index.html`.
+Just run command below to run the demo application and browse to `http://localhost:8080/swagger-ui/index.html`.
```

Comment:
Rocket is running in port `8000` by default, so there is no need to change this.